crates/router/src/core/payment_methods/cards.rs

```diff
@@ -4638,6 +4870,73 @@ async fn get_pm_list_context(
     Ok(payment_method_retrieval_context)
 }
 
+#[cfg(all(
+    feature = "v2",
+    feature = "payment_methods_v2",
+    feature = "customer_v2"
+))]
+async fn get_pm_list_context(
```

Comment:
Payment method related function should not be in cards.rs, instead it should be in payment_method.rs file